### PR TITLE
Draft of commands

### PR DESCRIPTION
**Pull request by seisman**
Related to #67.

### CODE REVIEW

File: source/computer/commands.rst [hunk 1]
```diff
@@ -1,2 +1,76 @@
...
+.. rubric:: 命令目录
+
+.. hlist::
+    :columns: 4
+
+    -   压缩与解压
+
+        - `tar`_
+
+    -   查看文件内容
```
Comment (core-man) [inline source/computer/commands.rst:16]:
```suggestion
    -   文件查看
```

File: source/computer/commands.rst [hunk 2]
```diff
@@ -1,2 +1,76 @@
...
+
+    -   文件处理
+
+        - `gawk`_
+        - `cut`_
+        - `grep`_
+        - `wc`_
+        - `sort`_
+        - `uniq`_
+
```
Comment (core-man) [inline source/computer/commands.rst:30]:
```suggestion

    -   文件搜索

        - `locate`_
        - `find`_

```

File: source/computer/commands.rst [hunk 3]
```diff
@@ -1,2 +1,76 @@
...
+    -   其他
```
Comment (core-man) [inline source/computer/commands.rst:31]:
```suggestion
. hlist::
    :columns: 4

    -   输出

        - `echo`_
        - `printf`_

    -   其他
```

File: source/computer/commands.rst [hunk 4]
```diff
@@ -1,2 +1,76 @@
...
+
+wc
+--
+
+sort
+----
+
+uniq
+----
+
```
Comment (core-man) [inline source/computer/commands.rst:68]:
```suggestion

echo
------


printf
-------

```

File: source/computer/commands.rst [hunk 5]
```diff
@@ -1,2 +1,76 @@
...
+wc
+--
+
+sort
+----
+
+uniq
+----
```
Comment (core-man) [inline source/computer/commands.rst:67]:
```suggestion
----

locate
--------

find
------

```

File: source/computer/commands.rst [hunk 6]
```diff
@@ -1,2 +1,100 @@
...
+    -   文件查看
+
+        - `cat`_
+        - `head`_
+        - `tail`_
+
+    -   文件处理
+
+        - `gawk`_
+        - `cut`_
```
Comment (seisman) [inline source/computer/commands.rst:26]: `cut` and `paste` are useful for formal data processing, but I think we're recommending performing data processing using Python, so `cut` and `paste` are less useful, right?

Comment (core-man) [reply source/computer/commands.rst:26]: Of course, we shoule use python. but do we want new babies kown them and use them if they just need to do some tests in CML?

Comment (seisman) [reply source/computer/commands.rst:26]: But why do they need to run or test `cut` in CLI?

Comment (core-man) [reply source/computer/commands.rst:26]: you mean we may use gawk instead?

Comment (seisman) [reply source/computer/commands.rst:26]: No, I mean we should use pure Python codes for data processing, no calling of `cut`, `paste`, `gawk` etc.

Comment (core-man) [reply source/computer/commands.rst:26]: I agree, but we may use them sometimes in teminal.

Comment (seisman) [reply source/computer/commands.rst:26]: I may use it once a year 😃

Comment (core-man) [reply source/computer/commands.rst:26]: > I may use it once a year 😃

me, too. please remove it or them.😅😅

File: source/computer/commands.rst [hunk 7]
```diff
@@ -1,2 +1,100 @@
...
+
+    -   文件搜索
+
+        - `locate`_
+        - `find`_
+
+    -   输出
+
+        - `echo`_
+        - `printf`_
```
Comment (seisman) [inline source/computer/commands.rst:39]: More thoughts:

1. Remove `echo`: I think we will mention it in Linux 101, when we talk about **PATH**
2. Remove `printf`: never used it unless writing shell scripts
3. Sort commands alphabetically?

Comment (core-man) [reply source/computer/commands.rst:39]: Yes to all.

**Comment by core-man** [comment]
We will work on this PR for all the commands, or we may just come to a TOC in this PR?

File: source/computer/commands.rst (hunk 7 above)
Comment (seisman) [reply source/computer/commands.rst]: Done in 747ef20826b56dff80ff698bf71dbfe1b1238493.

**Comment by seisman** [comment]
> We will work on this PR for all the commands, or we may just come to a TOC in this PR?

Just a TOC. Better to work on each common in separate PRs.

**Review by core-man — APPROVED** [review]

**Pull request merged**